tests/api/test_group_api.py

```diff
@@ -0,0 +1,46 @@
+import json
+import logging
+import unittest
+import requests
+from fauxfactory import gen_string
+from insights.config import Settings
+logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.DEBUG)
+
+
+class GroupApi(unittest.TestCase):
+    def setup_class(self):
+        self.setting = Settings()
+        self.session = requests.session()
+        self.session.cert = self.setting.get_certs()
+        self.session.verify = False
+        self.base_url = self.setting.get('api', 'url')
+
+    def test_group_api(self):
+        self.display_name = gen_string('alpha', 8)
+        # Create group
+        self.create_group = self.session.post(self.base_url + '/v1/groups',
+                                              data={'display_name': self.display_name})
+        assert self.create_group.status_code == 201
+        self.result = self.create_group.text
+        response = json.loads(self.result)
+        self.group_name = response['display_name']
+        assert self.group_name == self.display_name
+
+        # Request groups
+        self.request_groups = self.session.get(self.base_url + '/v1/groups/')
+        assert self.request_groups.status_code == 200
+
+        # Request group
+        self.group_id = response['id']
+        self.request_group = self.session.get(self.base_url + '/v1/groups/' + str(self.group_id))
+        assert self.request_group.status_code == 200
+        request_group_response = self.request_group.text
+        response_id = json.loads(request_group_response)
+        response_group_id = response_id['id']
+        assert response_group_id == self.group_id
+
+        # Delete group
+        self.delete_group = self.session.delete(self.base_url + '/v1/groups/' + str(self.group_id))
+        assert self.delete_group.status_code == 204
+        after_deleting_group = self.session.get(self.base_url + '/v1/groups/' + str(self.group_id))
+        assert after_deleting_group.status_code == 404
```

Comment:
add some loggers saying 'group deleted successfully' and a message with assert statement.